homeassistant/components/lektrico/strings.json

```diff
@@ -38,6 +38,17 @@
         "name": "Dynamic limit"
       }
     },
+    "select": {
+      "lb_mode": {
+        "name": "Lb mode",
```

Comment:
What Lb?

Reply:
It comes from Load balancing mode.
Our EM allows the user to select the source of the electricity used to charge vehicles.
The EM also commands the chargers how much to charge according to the available electricity.
When Lb Mode is Power:  EM allows charging from the grid and keeps charging as long as the breaker rating is not exceeded.
When Lb Mode is Green: EM allows charging only from solar panels.
When Lb Mode is Disabled: EM does not command the chargers, it only reports the consum.

Reply:
Should we just call it load balancing mode?

Reply:
I renamed lb mode with load balancing mode, thank you.